nautobot/utilities/templatetags/helpers.py

```diff
@@ -308,22 +309,28 @@ def get_docs_url(model):
     """Return the documentation URL for the specified model.
 
     Nautobot Core models have a path like docs/models/{app_label}/{model_name}
-    while plugins will have {app_label}/docs/models/{model_name}.
+    while plugins will have {app_label}/docs/models/{model_name}. If the html file
+    does not exist, this function will return None.
 
     Args:
         model (models.Model): Instance of a Django model
 
     Returns:
-        str: URL for the documentation of the object.
+        str: static URL for the documentation of the object.
+        or
+        None
 
     Example:
         >>> get_docs_url(obj)
         "static/docs/models/dcim/site.html"
     """
     if model._meta.app_label in settings.PLUGINS:
-        return f"{settings.STATIC_URL}{model._meta.app_label}/docs/models/{model._meta.model_name}.html"
+        path = f"{model._meta.app_label}/docs/models/{model._meta.model_name}.html"
+        if find(path):
+            return static(path)
+        return None
 
-    return f"{settings.STATIC_URL}docs/models/{model._meta.app_label}/{model._meta.model_name}.html"
+    return static(f"docs/models/{model._meta.app_label}/{model._meta.model_name}.html")
```

Comment:
Any reason not to have the same `find(path)` check for non-plugin models as well?

Reply:
No reason, but all of Nautobot models should have documentation already. I will add it in.